src/components/pages/Contact/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Content, Header } from 'antd/lib/layout/layout';
+import { Layout, Row, Col, Button, Space } from 'antd';
+import Navbar from '../../Navbar';
+import styles from './Contact.module.css';
+import logo from '../../../assets/orangelogo.png';
+
+export default function Contact() {
```

Comment:
All the code in here looks great, but anything that uses the Layout > Navbar > Content structure should be under the BaseLayout component to reduce re rendering and cut down on duplicate code. See if you can figure out how to add it there, im sure you can.